gerrit-entrypoint.sh

```diff
@@ -184,11 +180,40 @@ if [ "$1" = "/gerrit-start.sh" ]; then
   echo "Upgrading gerrit..."
   su-exec ${GERRIT_USER} java ${JAVA_OPTIONS} ${JAVA_MEM_OPTIONS} -jar "${GERRIT_WAR}" init --batch -d "${GERRIT_SITE}" ${GERRIT_INIT_ARGS}
   if [ $? -eq 0 ]; then
-    echo "Reindexing..."
-    su-exec ${GERRIT_USER} java ${JAVA_OPTIONS} ${JAVA_MEM_OPTIONS} -jar "${GERRIT_WAR}" reindex --verbose -d "${GERRIT_SITE}"
-    echo "Reindexing accounts..."
-    su-exec ${GERRIT_USER} java ${JAVA_OPTIONS} ${JAVA_MEM_OPTIONS} -jar "${GERRIT_WAR}" reindex --verbose --index accounts -d "${GERRIT_SITE}"
-    echo "Upgrading is OK."
+    GERRIT_VERSIONFILE="${GERRIT_SITE}/gerrit_version.txt"
+
+    if [ -z "${IGNORE_VERSIONCHECK}" ]; then
+      # dont perform a version check and never do a full reindex
+      NEED_REINDEX=0
```

Comment:
I'm confused by this.  When `-z "${IGNORE_VERSIONCHECK}"` is true, that means IGNORE_VERSIONCHECK isn't defined and let's use the default behavior, right? I think the default is re-index and `NEED_REINDEX=1`, isn't it?